packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -57,11 +108,13 @@ const NameType = (props) => {
       <PropertyType>
         {typeToRender.displayPrefix && typeToRender.displayPrefix}
 
-        {isRegex(props.property.name) && !typeToRender.displayPrefix
+        {getParameterType(props.property, typeToRender.type, props.apiKey)}
+
+        {/* {isRegex(props.property.name) && !typeToRender.displayPrefix
           ? `Any ${typeof typeToRender.type} property matching this regular expression`
           : typeof typeToRender.type === 'string'
           ? typeToRender.type
-          : typeToRender.type}
+          : typeToRender.type} */}
```

Comment:
I think we can safely get rid of this as it's been commented for quite a long time